event_sourcery-postgres.gemspec

```diff
@@ -6,15 +6,15 @@ require 'event_sourcery/postgres/version'
 Gem::Specification.new do |spec|
   spec.name          = 'event_sourcery-postgres'
   spec.version       = EventSourcery::Postgres::VERSION
-  # TODO: update authors
-  spec.authors       = ['Steve Hodgkiss']
-  spec.email         = ['steve@hodgkiss.me']
+
+  spec.authors       = ['Envato']
+  spec.email         = ['rubygems@envato.com']
```

Comment:
Perhaps we name contributors in a section of the readme in similar fashion to https://github.com/envato/double_entry/pull/131